### ISSUE TITLE
BulkIndexRequest can't store external version 0.

### ISSUE THREAD

**Issue by perj** [olivere/elastic#1057]
### Which version of Elastic are you using?

elastic.v6 (for Elasticsearch 6.x)

### Please describe the expected behavior

It should be possible to store a document with external version 0 in Elasticsearch. It works with the IndexService but not with BulkIndexRequest. We use a kafka topic as input and use the offset as version, which is how it came up.

### Please describe the actual behavior

The issue stems from the "omitempty" marker on the version field in BulkIndexRequest. The number 0 is thus skipped by the marshaller.

### Any steps to reproduce the behavior?

```
	req := elastic.NewBulkIndexRequest().Id("foo").VersionType("external").Version(0)
        fmt.Print(req)
```
should print
```
{"index":{"_id":"foo","version":0,"version_type":"external"}}
{}
```
actually prints
```
{"index":{"_id":"foo","version_type":"external"}}
{}
```

**Comment by olivere** [comment]
This should be fixed in 6.2.17.

**Comment by perj** [comment]
Cool, thanks a lot.
